11-query/04-delete-query.md

```diff
@@ -46,39 +51,81 @@ delete $emp;
 GraqlDelete query = Graql.match(
   var("org").isa("organisation").has("name", "Pharos"),
   var("emp").isa("employment").rel("employer", "org").rel("employee", "p")
-).delete("emp");
+).delete(var("emp").isa("employment"));
 ```
 [tab:end]
 </div>
 
 This deletes all instances of the `employment` type where the `employer` is an `organisation` with `name` of `"Pharos"`.
 
-## Delete Associations with Attributes
-Attributes with the same value are shared among their owners. It's important that one understands thoroughly how [attributes are defined](../09-schema/01-concepts.md#define-an-attribute) in a Grakn knowledge graph prior to performing `delete` queries on them.
+Attributes are normally owned by other concepts (ie. `$x has attribute $a`). This means we can both delete an attribute
+ownership, as in the next section, or delete the instance itself following the above style of using `isa` in the delete clause.
+
+## Delete Attribute Ownerships
+We can remove the ownership of an attribute by another concept in the same way it is inserted or queried: using the `has` clause.
 
-To delete only the association that a thing has with an attribute, we use the `via` keyword to capture and delete the relation between the owner and the owned attribute - NOT the instance of the attribute type itself, as doing so disowns the instance from any other instance that may have owned it.
+Note that attributes with the same value are shared among their owners, so usually shouldn't delete the attribute directly. 
```

Comment:
> so usually shouldn't delete the attribute directly.

reads a bit informal. how about "Note that attributes with the same value are shared among their owners. For this reason, usually, an attribute is not deleted directly."?